bin/exec.go

```diff
@@ -39,45 +40,44 @@ func checkStatusListForExp(status []v1alpha1.ExperimentStatuses, jobName string)
 	return -1
 }
 
-var kubeconfig string
-var err error
-var config *rest.Config
-
 func main() {
 
 	engineDetails := utils.EngineDetails{}
-
+	clients := utils.ClientSets{}
 	// Getting the config
 	config, err := getKubeConfig()
 	if err != nil {
 		log.Info("Error in fetching the config")
 		log.Infoln(err.Error())
 	}
 
-	engineDetails.Config = config
-
-	// Genrationg Client Set for more functionality
-	var clients utils.ClientSets
-
-	// ClientSet Generation
-	clients.KubeClient, clients.LitmusClient, err = utils.GenerateClientSets(engineDetails.Config)
+	// clientSet creation for further use.
+	kClient, lClient, err := utils.GenerateClientSets(config)
 	if err != nil {
-		log.Info("Unable to generate ClientSet while Creating Job")
-		return
+		log.Infof("Unable to create ClientSets")
 	}
-
+	clients.KubeClient = kClient
+	clients.LitmusClient = lClient
+	//var perExperiment utils.ExperimentDetails
+	//perExperiment.Namespace = engineDetails.AppNamespace
 	// Fetching all the ENV's needed
 	utils.GetOsEnv(&engineDetails)
 	log.Infoln("Experiments List: ", engineDetails.Experiments, " ", "Engine Name: ", engineDetails.Name, " ", "appLabels : ", engineDetails.AppLabel, " ", "appNamespace: ", engineDetails.AppNamespace, " ", "appKind: ", engineDetails.AppKind, " ", "Service Account Name: ", engineDetails.SvcAccount)
 
 	// Steps for each Experiment
 	for i := range engineDetails.Experiments {
 
+		perExperiment := utils.NewExperimentDetails()
+		perExperiment.Name = engineDetails.Experiments[i]
+		perExperiment.Namespace = engineDetails.AppNamespace
+		perExperiment.SvcAccount = engineDetails.SvcAccount
+		log.Infof("printing perExperiment.Name: %v, perexperiment.Namespace : %v", perExperiment.Name, perExperiment.Namespace)
```

Comment:
Can you verify the log format, Is it correct!!

Reply:
Yes

Reply:
okay!! Then can you make it like this:
```
log.Infof("experiment name: %s, experiment namespace : %s", perExperiment.Name, perExperiment.Namespace)
```